### PR TITLE
qemu: update qemu to 4.1

### LINKED ISSUE #120: Update QEMU to 4.1
http://lists.nongnu.org/archive/html/qemu-devel/2019-04/msg03851.html

### PR DESCRIPTION
**Pull request by karpa4o4**
Close #120

### CODE REVIEW

File: qemu/Dockerfile [hunk 1]
```diff
@@ -104,7 +104,6 @@ RUN echo http://mirror.yandex.ru/mirrors/alpine/v3.7/community >> /etc/apk/repos
             mips64el-softmmu \
             moxie-softmmu \
             ppc-softmmu \
-            ppcemb-softmmu \
```
Comment (karpa4o4) [inline qemu/Dockerfile]: The "qemu-system-ppcemb" target has been removed. "qemu-system-ppc" or "qemu-system-ppc64" should be used instead.

Comment (eugulixes) [reply qemu/Dockerfile]: Add a proof link please.

Comment (karpa4o4) [reply qemu/Dockerfile]: https://wiki.qemu.org/ChangeLog/3.1#Incompatible_changes

File: qemu/Dockerfile [hunk 2]
```diff
@@ -129,7 +128,7 @@ RUN echo http://mirror.yandex.ru/mirrors/alpine/v3.7/community >> /etc/apk/repos
         --enable-kvm \
         --enable-libiscsi \
         --enable-libnfs \
-        --enable-libssh2 \
+        --enable-libssh \
```
Comment (karpa4o4) [inline qemu/Dockerfile]: Proof: https://wiki.qemu.org/ChangeLog/4.1#Build_Dependencies

File: qemu/Dockerfile (hunk 1 above)
Comment (eugulixes) [reply qemu/Dockerfile]: You have to remove it from README as well.

File: qemu/Dockerfile (hunk 2 above)
Comment (eugulixes) [reply qemu/Dockerfile]: Let's add the link before the line.

**Pull request merged**